frontend/dashboard/pages/PageLayout/DashboardHeader/SmallHeaderMenu/SmallHeaderMenu.test.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { SmallHeaderMenu } from './SmallHeaderMenu';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import userEvent from '@testing-library/user-event';
+import { headerContextValueMock } from '../../../../testing/headerContextMock';
+import { HeaderMenuGroupKey } from '../../../../enums/HeaderMenuGroupKey';
+import { HeaderContext, type HeaderContextProps } from '../../../../context/HeaderContext';
+import { renderWithProviders } from '../../../../testing/mocks';
+import { addFeatureFlagToLocalStorage, FeatureFlag } from 'app-shared/utils/featureToggleUtils';
+
+describe('SmallHeaderMenu', () => {
+  beforeAll(() => {
+    addFeatureFlagToLocalStorage(FeatureFlag.OrgLibrary);
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render the menu trigger button with the correct text', () => {
+    renderSmallHeaderMenu();
+
+    expect(screen.getByRole('button', { name: textMock('top_menu.menu') })).toBeInTheDocument();
```

Comment:
This is a repeating query and can be made into a function, like this:

```tsx
function getTopMenuButton() {
  return screen.getByRole('button', {
    name: textMock('top_menu.menu'),
  });
}
```

and be used like this:
```suggestion
    expect(getTopMenuButton()).toBeInTheDocument();
```

This will apply to multiple of these tests.